code/modules/language/pohlan.dm

```diff
@@ -0,0 +1,14 @@
+/datum/language/pohlan
+	name = "Poh'lan"
+	desc = "The language of the Grod people. It is synthesized using chitters and grumbles, and is completely incomprehensible."
+	icon_state = "pohlan"
+	speech_verb = "rumbles"
+	exclaim_verb = "decrees"
+	sing_verb = "harks"
+	key = "e"
+	sentence_chance = 8
+	space_chance = 10
+	syllables = list(
+		 "Ży", "zy", "Zi", "rzy", "szy", "ów", "rze", "że", "ść", "rzsz", "sz", "aż", "cz",
+		 "czy", "cze", "ci", "cin", "dze", "dzi", "drzy", "żdż", "dzy", "bź", "Ić", "ąż", "ęż",
```

Comment:
I am not sure if adding polish characters is a great idea
@PowerfulBacon

Reply:
![image](https://user-images.githubusercontent.com/40559528/185058769-f69d0343-0d43-4427-8d0c-b3af0983a4a8.png)